static/src/javascripts/projects/common/modules/experiments/tests/membership-message.js

```diff
@@ -1,35 +1,42 @@
 define([
     'common/utils/config',
     'common/utils/detect',
+    'common/utils/storage',
     'common/utils/template',
-    'common/modules/ui/message',
     'common/modules/identity/api',
-    'text!common/views/memebership-message.html'
+    'common/modules/ui/message',
+    'text!common/views/membership-message.html'
 ], function (
     config,
     detect,
+    storage,
     template,
-    Message,
     idApi,
-    messageTmpl
+    Message,
+    messageTemplate
 ) {
 
     return function () {
 
         this.id = 'MembershipMessage';
-        this.start = '2015-06-08';
-        this.expiry = '2015-06-19';
+        this.start = '2015-06-17';
+        this.expiry = '2015-06-26';
         this.author = 'David Rapson';
-        this.description = 'Test if logged in users are encouraged to join Membership';
-        this.audience = 1;
-        this.audienceOffset = 0;
-        this.successMeasure = 'Users will be interested in Membership';
-        this.audienceCriteria = '100% of logged in users, only on article pages';
+        this.description = 'Test if loyal users are encouraged to join Membership as a Supporter';
+        this.audience = 0.5;
+        this.audienceOffset = 0.5;
+        this.successMeasure = 'Loyal users will be interested in becoming a Supporter';
+        this.audienceCriteria = 'Users who have seen at least 10 pages, message shown only on article pages';
         this.dataLinkNames = 'supporter message, hide, read more';
-        this.idealOutcome = 'Users will sign up as a Guardian Member';
+        this.idealOutcome = 'Users will sign up as a Supporter';
 
         this.canRun = function () {
-            return idApi.isUserLoggedIn() && config.page.contentType === 'Article';
+            /**
+             * Exclude adblock users to avoid conflicts with similar adblock Supporter message,
+             * only show to users who have viewed at least 10 pages.
+             */
+            var alreadyVisited = storage.local.get('alreadyVisited') || 0;
```

Comment:
@stephanfowler Ended up using the `alreadyVisited ` counter that's being set as part of the adblock check, see: https://github.com/guardian/frontend/blob/0d5c02d2fbf6a70fc77f9cf1638db917d244bc31/static/src/javascripts/bootstraps/common.js#L280 . It's currently slightly coupled to adblock but is incremented globally as part of common bootstrap so seems OK to use in this case.

Using the popular tags didn't seem ideal in the end as an article can have a variable number of tags, so that list can seemingly be large even when the user has only visited one article.